### PR TITLE
fixes #9397 - Make usage of @host.operatingsystem consistent

### PR DESCRIPTION
**Pull request by johscheuer**
Make usage of @host.operatingsystem in UnattendedController consistent to looklike "@mediapath = @host.operatingsystem.mediumpath @host" in all definitions. [Issue](http://projects.theforeman.org/issues/9397)

### CODE REVIEW

**Comment by dLobatog** [comment]
[test] because of the latest enabled cops

**Comment by johscheuer** [comment]
The failed tests are not associated with this PR?

File: app/controllers/unattended_controller.rb [hunk 1]
```diff
@@ -182,7 +182,7 @@ def set_content_type
 
   def load_template_vars
     # load the os family default variables
-    send "#{@host.os.pxe_type}_attributes"
+    send "#{@host.operatingsystem.pxe_type}_attributes"
```
Comment (johscheuer) [inline app/controllers/unattended_controller.rb:184]: Many files use `@host.os` and `@host.operatingsystem`. I would prefer to use only one alias/attribute per file to make it cleaner.

What your thoughts about this?

**Comment by domcleal** [comment]
[test]

File: app/controllers/unattended_controller.rb (hunk 1 above)
Comment (domcleal) [inline app/controllers/unattended_controller.rb:184]: Sounds sane!

Comment (johscheuer) [inline app/controllers/unattended_controller.rb:184]: Would you suggest to create a new PR for this or merge it with this PR? Also what would you like more `host.os` or `host.operatingsystem`? I would prefer `host.operatingsystem`.

Comment (domcleal) [inline app/controllers/unattended_controller.rb:184]: Doing it all in one PR is fine.  I don't really mind which, maybe @eLobato has an opinion?

**Comment by dLobatog** [comment]
@johscheuer Thanks, I'll put this in. You might want to write a [custom cop](https://github.com/bbatsov/rubocop#custom-cops) or something like that, otherwise math tells us this consistency will go away eventually. 

Merged as f23483a2eda81e9b1a2cbf1cd1c26bb470fa0809 :bomb:

**Comment by johscheuer** [comment]
@eLobato Thanks, I will have a look

**Pull request closed without merging**